csv-utils/userToCsv.js

```diff
@@ -0,0 +1,43 @@
+exports.userToCsvHeader = (users) => {
+  const convertedUsers = [];
+
+  for (let i = 0; i < users.length; i += 1) {
+    const user = users[i];
+    const { Username, Attributes, UserLastModifiedDate } = user;
+    console.log(user);
+
+    const dateTime = new Date(UserLastModifiedDate);
+
+    // Update this to transfer whatever attributes are needed from the previous user pool to the new pool
+
+    if (Attributes[1].Value === 'true') {
+      convertedUsers.push([
+        '',
```

Comment:
I believe a comment here about the rigid structure of the expected CVS would be helpful.
Also, I would love to see a line-by-line comment for what each of these strings represents (that might be pretty tedious though...)

Reply:
Added in a comment